### ISSUE TITLE
Composer update fails

### ISSUE THREAD

**Issue by ghost** [nexylan/slack#45]
Using command:
```
 composer require nexylan/slack php-http/guzzle6-adapter
```

Gets this:

```
  Problem 1
    - Conclusion: don't install nexylan/slack 2.x-dev
    - Installation request for php-http/guzzle6-adapter ^2.0@dev -> satisfiable by php-http/guzzle6-adapter[2.x-dev, v2.0.0].
    - Installation request for nexylan/slack ^2.0@dev -> satisfiable by nexylan/slack[2.x-dev, v2.0.0].
    - nexylan/slack v2.0.0 requires php-http/client-common ^1.7 -> satisfiable by php-http/client-common[1.7.0, 1.8.0, 1.8.1, 1.8.x-dev].
    - php-http/client-common 1.7.0 requires php-http/httplug ^1.1 -> satisfiable by php-http/httplug[v1.1.0].
    - php-http/client-common 1.8.0 requires php-http/httplug ^1.1 -> satisfiable by php-http/httplug[v1.1.0].
    - php-http/client-common 1.8.1 requires php-http/httplug ^1.1 -> satisfiable by php-http/httplug[v1.1.0].
    - php-http/client-common 1.8.x-dev requires php-http/httplug ^1.1 -> satisfiable by php-http/httplug[v1.1.0].
    - Conclusion: don't install php-http/httplug v1.1.0
```

**Comment by soullivaneuh** [comment]
Hi @azcoppen,

The library seems to not be compatible with guzzle6-adapter 2.0.

You can solve it with this require command:

```
composer require nexylan/slack php-http/guzzle6-adapter:^1.0
```

Would you like to work on PR to make it compatible?

**Comment by ghost** [comment]
I'm good man, thanks. Had to rewrite it directly in Guzzle and remove the package unfortunately.

**Comment by soullivaneuh** [comment]
Understand, but I reopen the issue. It's still accurate.

**Comment by soullivaneuh** [comment]
Closed in favor of #54
